Cargo.toml

```diff
@@ -19,19 +19,21 @@ exclude = [".github/workflows/ci.yml", "github/workflows/rust.yml",
 ]
 
 [dependencies]
-blake2 = "0.9"
 dusk-bytes = "0.1"
 dusk-bls12_381 = {version="0.6", default-features=false}
-subtle = {version="^2.3", default-features = false}
-rand_core = {version = "0.5", default-features=false}
+subtle = {version="2.0", default-features = false}
```

Comment:
Care to elaborate this downgrade…?
What are we gaining, what are we losing?

Reply:
Nevermid. Will bring it back.